src/edfi-paging-test/edfi_paging_test/__main__.py

```diff
@@ -45,20 +49,38 @@ def _configure_logging(configuration: MainArguments) -> None:
     _redefine_debug_as_verbose(configuration, "urllib3.connectionpool")
 
 
+def _url_reachable(url):
```

Comment:
@JBrenesSimpat could you move this to logic to a helper class and call the method from inside the 'run' method?